recipe/conda_build_config.yaml

```
@@ -1,5 +1,8 @@
-c_stdlib_version:  # [unix]
-  - 2.17           # [linux]
+c_stdlib:          # [linux64]
+  - sysroot        # [linux64]
+
+c_stdlib_version:  # [linux64 or (osx and x86_64)]
+  - 2.17           # [linux64]
```

we don't need all that, since the baseline is now 2.17
```suggestion
c_stdlib_version:  # [osx and x86_64]
```